Source/WebGPU/WGSL/AST/ASTTypeDecl.h

```diff
@@ -27,12 +27,18 @@
 
 #include "ASTExpression.h"
 
+#include <wtf/ThreadSafeRefCounted.h>
 #include <wtf/TypeCasts.h>
 #include <wtf/text/WTFString.h>
 
-namespace WGSL::AST {
+namespace WGSL {
+class ResolveTypeReferences;
 
-class TypeDecl : public Node {
+namespace AST {
+
+class StructDecl;
+
+class TypeDecl : public Node, public ThreadSafeRefCounted<TypeDecl> {
```

Comment:
Threadsafe? Really?

Reply:
I guess it was force of habit. I'll remove the ThreadSafe.